workflows/patch_workflow.ipynb

```diff
@@ -0,0 +1,202 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Workflows offer a versatile framework for constructing your inference pipeline. It equips you with the fundamental components for sophisticated machine learning ensemble modeling and incorporating business logic.\n",
+    "\n",
+    "Leverage Workflows to handle intricate processes, extract valuable insights, and construct tailored solutions. With Workflows, your machine learning models can seamlessly function as interconnected nodes within a graph.\n",
+    "\n",
+    "Clarifai simplifies the process of integrating diverse models, enabling you to execute intricate data operations and design solutions tailored to your precise business requirements.\n",
+    "\n",
+    "Utilize workflows to link multiple models in a sequence, steering and influencing their behavior effectively."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation and Setup"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is required to set **CLARIFAI_PAT** as Environment Variable.\n",
+    "\n",
+    "More info on [Personal Access Tokens](https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install clarifai"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"CLARIFAI_PAT\"] = \"CLARIFAI_PAT\" # replace with your own PAT key here\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### For Colab\n",
+    "\n",
+    "To access config yamls from examples repo, you can clone the repo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!git clone https://github.com/Clarifai/examples.git\n",
+    "%cd /content/examples/workflows/"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Patching Workflow\n",
+    "\n",
+    "To make rapid adjustments to existing Clarifai community workflows using a custom yml configuration or update some of the workflow details (description, notes..etc), the SDK offers a patch feature. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from clarifai.client.user import User\n",
+    "# Create a new app (replace USER_ID and APP_ID with your own)\n",
+    "app = User(user_id=\"USER_ID\").create_app(app_id=\"APP_ID\", base_workflow=\"Universal\")"
+   ]
+  },
+  {
```

Comment:
Can we add small notes before this patch cells like explaining what we are doing with the patch operation.

Reply:
Done